# Exploración de datos PAES 2026

En este notebook realzaremos una primera revisión de los conjuntos de datos verificados en el paso `00_verificacion_entorno`. El objetivo es conocer la estructura y contenido en terminos de datos de cada fuente, utilizando como referencia el archivo PDF que acompaña a cada fuente y que entrega, entre otros, información sobre: observaciones según proceso de admisión, responsable del dato en particular, Variables (nombre, tipo, descripción y valores) asi como también códigos asociados a las unidades territoriales. 

Para esta experiencia se toma como referencia los contenidos de clases, evaluando en cada conjunto lo siguiente: 

- dimensiones del conjunto
- primeras observaciones
- variables y tipos de datos
- valores faltantes
- posibles duplicados
- variables cualitativas y cuantitativas relevantes
- posibles identificadores o llaves
- unidad de observación de cada tabla

El proceso se realizará con cada archivo, aun cuando esto podria hacerse de forma paralela. La idea es, apoyados en el PDF de cada archivo, ir generando un dominio acabado de cada dataset.  

La forma de trabajo definida utiliza los archivos disponibles en `data/raw` como referencia. Durante el proceso de análisis exploratorio y/o transformacion, los datasets quedaran en `data/intermediate`, en caso de que se requiera complementar o realizar algún paso adicional. Finalmente, el conjunto preparado y listo para análisis quedara en `data/processed`

In [2]:
from pathlib import Path
import pandas as pd

DIRECTORIO_ACTUAL = Path.cwd()
DIRECTORIO_RAIZ = DIRECTORIO_ACTUAL.parent
RAW_DIR = DIRECTORIO_RAIZ / "data" / "raw"


## Inscritos y Puntajes

Comenzaremos con el conjunto `A_INSCRITOS_PUNTAJES_PAES_2026_PUB_MRUN.csv`

In [3]:
RUTA_INSCRITOS = RAW_DIR / "PAES-2026-Inscritos-Puntajes" / "A_INSCRITOS_PUNTAJES_PAES_2026_PUB_MRUN.csv" #Definimos CARPETA / Archivo

# Primero miramos el archivo RAW. Si bien podriamos haber definido un comando para leer ya formateado el archivo, la idea es 
# poder entender en operación que esperar para definir un metodo de lectura. 
with open(RUTA_INSCRITOS, "r", encoding="utf-8") as archivo:
    for _ in range(3):
        print(archivo.readline())


MRUN;ANYO_PROCESO;COD_SEXO;FECHA_NACIMIENTO;RBD;CODIGO_ENS;LOCAL_EDUCACIONAL;UNIDAD_EDUCATIVA;NOMBRE_UNIDAD_EDUC;RAMA_EDUCACIONAL;DEPENDENCIA;CODIGO_REGION_EGRESO;NOMBRE_REGION_EGRESO;CODIGO_PROVINCIA_EGRESO;NOMBRE_PROVINCIA_EGRESO;CODIGO_COMUNA_EGRESO;NOMBRE_COMUNA_EGRESO;ANYO_DE_EGRESO;PROMEDIO_NOTAS;PTJE_NEM;PORC_SUP_NOTAS;PTJE_RANKING;CLEC_REG_ACTUAL;MATE1_REG_ACTUAL;MATE2_REG_ACTUAL;HCSOC_REG_ACTUAL;CIEN_REG_ACTUAL;CLEC_INV_ACTUAL;MATE1_INV_ACTUAL;MATE2_INV_ACTUAL;HCSOC_INV_ACTUAL;CIEN_INV_ACTUAL;CLEC_REG_ANTERIOR;MATE1_REG_ANTERIOR;MATE2_REG_ANTERIOR;HCSOC_REG_ANTERIOR;CIEN_REG_ANTERIOR;CLEC_INV_ANTERIOR;MATE1_INV_ANTERIOR;MATE2_INV_ANTERIOR;HCSOC_INV_ANTERIOR;CIEN_INV_ANTERIOR;PROMEDIO_CM_MAX;CLEC_MAX;MATE1_MAX;MATE2_MAX;HCSOC_MAX;CIEN_MAX;RINDIO_PROCESO_ANTERIOR;RINDIO_PROCESO_ACTUAL;HABILITACION_POST;PRUEBA_REG_CL;FORMA_REG_CL;CORRECTAS_REG_CL;ERRADAS_REG_CL;OMITIDAS_REG_CL;PRUEBA_REG_M1;FORMA_REG_M1;CORRECTAS_REG_M1;ERRADAS_REG_M1;OMITIDAS_REG_M1;PRUEBA_REG_M2;FORMA_REG_M2;CO

El resultado del codigo nos muestra que el separador utilizado en el .csv es `;` según el registro `MRUN;ANYO_PROCESO;COD_SEXO;FECHA_NACIMIENTO;RBD;`
En general uno podria asumir que para estos archivos .csv siempre el separador será `;` pero es una buena practica revisar el texto raw antes de declararlo en lectura como `viajes = pd.read_csv("viajes.csv", sep=";", decimal=",", low_memory=False)`


In [4]:
inscritos = pd.read_csv (RUTA_INSCRITOS, sep = ";", decimal= ",", low_memory=False)
inscritos

,MRUN,ANYO_PROCESO,COD_SEXO,FECHA_NACIMIENTO,RBD,CODIGO_ENS,LOCAL_EDUCACIONAL,UNIDAD_EDUCATIVA,NOMBRE_UNIDAD_EDUC,RAMA_EDUCACIONAL,...,PRUEBA_INV_CQUI,FORMA_INV_CQUI,CORRECTAS_INV_CQUI,ERRADAS_INV_CQUI,OMITIDAS_INV_CQUI,PRUEBA_INV_CTP,FORMA_INV_CTP,CORRECTAS_INV_CTP,ERRADAS_INV_CTP,OMITIDAS_INV_CTP
0,89,2026,2,200708,2308,310,1326,1,COLEGIO SAN ANTONIO DEL BALUARTE,H1,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
1,258,2026,1,200704,12305,310,2157,1,COLEGIO ALEXANDER FLEMING,H1,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
2,259,2026,2,200704,8005,510,2701,7,LICEO AYTUE,T2,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
3,496,2026,2,200711,10967,510,60,3,LICEO ELEUTERIO RAMIREZ MOLINA,T2,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
4,538,2026,2,200711,26538,310,4041,1,COLEGIO TERRAUSTRAL OESTE S.A,H1,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
320082,28540477,2026,1,200712,4708,410,1658,5,LICEO POLIVALENTE LAS SALINAS,T1,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
320083,28540478,2026,2,200705,218,610,61,4,LICEO FRANCISCO DE AGUIRRE,T3,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
320084,28540479,2026,2,200805,9900,610,836,5,COLEGIO BICENTENARIO DE EXCELENCIA POLITECNICO...,T3,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
320085,28540480,2026,1,200802,3110,310,6906,1,COMPLEJO EDUCACIONAL MAULE,H1,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0


Con la tabla anterior de resultado, ya tenemos un dataframe conceptualmente reconocible, por lo que procedemos a realizar las exploraciones basicas revisadas en clases. 

In [5]:
inscritos.shape # Para conocer filas, columnas

(320087, 131)

In [6]:
inscritos.head() # Para conocer las primeras filas, aunque esto de alguna forma ya lo obtuvimos al mostrar inscritos

,MRUN,ANYO_PROCESO,COD_SEXO,FECHA_NACIMIENTO,RBD,CODIGO_ENS,LOCAL_EDUCACIONAL,UNIDAD_EDUCATIVA,NOMBRE_UNIDAD_EDUC,RAMA_EDUCACIONAL,...,PRUEBA_INV_CQUI,FORMA_INV_CQUI,CORRECTAS_INV_CQUI,ERRADAS_INV_CQUI,OMITIDAS_INV_CQUI,PRUEBA_INV_CTP,FORMA_INV_CTP,CORRECTAS_INV_CTP,ERRADAS_INV_CTP,OMITIDAS_INV_CTP
0,89,2026,2,200708,2308,310,1326,1,COLEGIO SAN ANTONIO DEL BALUARTE,H1,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
1,258,2026,1,200704,12305,310,2157,1,COLEGIO ALEXANDER FLEMING,H1,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
2,259,2026,2,200704,8005,510,2701,7,LICEO AYTUE,T2,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
3,496,2026,2,200711,10967,510,60,3,LICEO ELEUTERIO RAMIREZ MOLINA,T2,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0
4,538,2026,2,200711,26538,310,4041,1,COLEGIO TERRAUSTRAL OESTE S.A,H1,...,Ciencias Química.,0,0,0,0,Ciencias Técnico Profesional.,0,0,0,0


Para conocer el detalle de las columnas utilizamos .info(). Dado que el archivo contiene 131 columnas es bastante probable que la funcion retorne un cuadro de resumen basico.

In [7]:
inscritos.info() 

<class 'pandas.DataFrame'>
RangeIndex: 320087 entries, 0 to 320086
Columns: 131 entries, MRUN to OMITIDAS_INV_CTP
dtypes: float64(2), int64(79), str(50)
memory usage: 319.9 MB


Por lo anterior, y como nosotros esperamos obtener información mas precisa, utilizamos la funcion con el argumento: 
verbose=True para forzar mostrar el detalle completo 
show_counts=True para mostrar el conteo de valores no nulos.

In [8]:
inscritos.info(verbose=True) 

<class 'pandas.DataFrame'>
RangeIndex: 320087 entries, 0 to 320086
Data columns (total 131 columns):
 #    Column                   Dtype  
---   ------                   -----  
 0    MRUN                     int64  
 1    ANYO_PROCESO             int64  
 2    COD_SEXO                 int64  
 3    FECHA_NACIMIENTO         int64  
 4    RBD                      str    
 5    CODIGO_ENS               str    
 6    LOCAL_EDUCACIONAL        str    
 7    UNIDAD_EDUCATIVA         str    
 8    NOMBRE_UNIDAD_EDUC       str    
 9    RAMA_EDUCACIONAL         str    
 10   DEPENDENCIA              str    
 11   CODIGO_REGION_EGRESO     str    
 12   NOMBRE_REGION_EGRESO     str    
 13   CODIGO_PROVINCIA_EGRESO  str    
 14   NOMBRE_PROVINCIA_EGRESO  str    
 15   CODIGO_COMUNA_EGRESO     str    
 16   NOMBRE_COMUNA_EGRESO     str    
 17   ANYO_DE_EGRESO           str    
 18   PROMEDIO_NOTAS           float64
 19   PTJE_NEM                 int64  
 20   PORC_SUP_NOTAS           int64  
 2

In [9]:
inscritos.info(verbose=True,show_counts=True) 

<class 'pandas.DataFrame'>
RangeIndex: 320087 entries, 0 to 320086
Data columns (total 131 columns):
 #    Column                   Non-Null Count   Dtype  
---   ------                   --------------   -----  
 0    MRUN                     320087 non-null  int64  
 1    ANYO_PROCESO             320087 non-null  int64  
 2    COD_SEXO                 320087 non-null  int64  
 3    FECHA_NACIMIENTO         320087 non-null  int64  
 4    RBD                      320087 non-null  str    
 5    CODIGO_ENS               320087 non-null  str    
 6    LOCAL_EDUCACIONAL        320087 non-null  str    
 7    UNIDAD_EDUCATIVA         320087 non-null  str    
 8    NOMBRE_UNIDAD_EDUC       316887 non-null  str    
 9    RAMA_EDUCACIONAL         319624 non-null  str    
 10   DEPENDENCIA              320087 non-null  str    
 11   CODIGO_REGION_EGRESO     320087 non-null  str    
 12   NOMBRE_REGION_EGRESO     318876 non-null  str    
 13   CODIGO_PROVINCIA_EGRESO  320087 non-null  str    
 14

La elección final de qué alternativa utilizar depende mayormente del tipo de dataset a revisar. 
 Un caso donde `show_counts=True` cobra valor, es por ejemplo en una encuesta, donde podrían existir valores `null`(vacíos) indicando que una persona no contestó una pregunta. 
 En este dataset en particular vemos que algunas columnas como `NOMBRE_UNIDAD_EDUC`, `RAMA_EDUCACIONAL` o `NOMBRE_REGION_EGRESO`, muestran menos valores no nulos respecto de las observaciones totales, lo cual será revisado en profundidad mas adelante. 

Un detalle particular en esta primera mirada es la definición del tipo de dato de `FECHA_NACIMIENTO`. En el archivo PDF se declara como una cadena o string (str) con formato AAAAMM pero en la info entregada por pandas, se declara como int64 (entero de 64 bits con signo), es decir, un numero. Si bien esto no necesariamente es un error de codificación o lectura, este tipo de detalles son relevantes a la hora de utilizar la información. Por lo que, una vez definido el conjunto de variables que efectivamente se utilizará, se deberán realizar las transformaciones necesarias para tratar de forma correcta la información contenida en cada variable. 

# Revisiones BASE

En un df con 131 columnas, hacer una revision estadistica de cada una parece demasiado, considerando aun la gran cantidad de trabajo que hace el codigo. 
Por lo tanto, partiremos con algunos checks que tienen sentido e iremos definiendo a medida que el trabajo avance, la pertinencia de columnas a analizar en el tiempo. 

La primera duda surge con el identificador MRUN que deberia ser unico por estudiante y considerando los 320087 registros, deberiamos tener esa cantidad de estudiantes. 

In [10]:
# Utilizando entonces la funcion .nunique() validaremos si esta hipotesis es correcta. 
inscritos["MRUN"].nunique() # Usamos ["MRUN"] para definir la columna en particular.
# El resultado es consistente. 


320087

In [11]:
# Uno podria para este mismo objetivo utilizar tambien .duplicated() con .sum() que suma los duplicados. 
inscritos["MRUN"].duplicated().sum()
# Dado que el resultado es 0, podemos afirmar con las dos formas declaradas, que cada fila corresponde a un alumno. 

np.int64(0)

Otro de los valores iniciales del DF es ANYO_PROCESO. Se podría asumir, dado que los datos corresponden al Proceso de Admisión 2026, que en esta fila deberían ser todos 2026. Lo corroboramos con value_counts() que nos entregara la cantidad de registros donde ANYO_PROCESO es 2026. Si hubiera un 2025 o cualquier otro valor, nos mostrara tambien ese resumen. 

In [12]:
inscritos["ANYO_PROCESO"].value_counts()
# El resultado es consistente, ANYO_PROCESO solo considera 2026

ANYO_PROCESO
2026    320087
Name: count, dtype: int64

## Revisión de variables asociadas a la formulación del proyecto

Considerando la pregunta planteada en el proceso de formulacion del proyecto capstone, se identifican un conjunto inicial de variables para el EDA. 
Para ello, definiremos un arreglo/lista, de variables del proyecto a modo de acotar el dataset inicial y comenzar a trabajar en las variables a analizar. 

In [23]:
variables_trabajo = [
    "MRUN", 
    "CLEC_REG_ACTUAL", 
    "MATE1_REG_ACTUAL", 
    "MATE2_REG_ACTUAL",
    "DEPENDENCIA",
    "RAMA_EDUCACIONAL", 
    "CODIGO_REGION_EGRESO",
    "NOMBRE_REGION_EGRESO"
]
inscritos[variables_trabajo].info()
# En esta funcion, tomamos inscritos que fue nuestra "varibale" asociada al dataframe que lee desde el .csv y seleccionamos
# el conjunto de variables/columnas que definimos en variables_trabajo y sobre esto aplicamos info()

# En esta oportunidad se ha dejado fuera columnas que no agregan valor dada la formulacion del proyecto, en esta etapa
# de todas formas, lo que hacemos con las listas o df, es seleccionar para trabajar, nunca eliminar, por lo que de ser 
# necesario otros datos en el futuro, se pueden agregar sin problemas. 

# En la primera ejecucion de esta instrucción, olvide colocar MATE2_REG_ACTUAL pero basto identificar esta condicion 
# para simplemente agregarla a las variables de trabajo y luego el codigo se ejecuta de forma transparente, sin mayor 
# impacto. Si en vez de utilizar una lista/arreglo, hubieramos definido las variables una por una, tendriamos que editar
# agregar hacia abajo cualquier instruccion que utilizara alguna variable que dejamos fuera. 

<class 'pandas.DataFrame'>
RangeIndex: 320087 entries, 0 to 320086
Data columns (total 8 columns):
 #   Column                Non-Null Count   Dtype
---  ------                --------------   -----
 0   MRUN                  320087 non-null  int64
 1   CLEC_REG_ACTUAL       320087 non-null  str  
 2   MATE1_REG_ACTUAL      320087 non-null  str  
 3   MATE2_REG_ACTUAL      320087 non-null  str  
 4   DEPENDENCIA           320087 non-null  str  
 5   RAMA_EDUCACIONAL      319624 non-null  str  
 6   CODIGO_REGION_EGRESO  320087 non-null  str  
 7   NOMBRE_REGION_EGRESO  318876 non-null  str  
dtypes: int64(1), str(7)
memory usage: 19.5 MB


Como resultado, obtenemos el resumen que nos entrega pandas de las variables seleccionadas, logrando el total de los registros en la mayoria de variables, excepto en 2. RAMA_EDUCACIONAL y CODIGO_REGION_EGRESO

In [24]:
# Para un análisis mas en profundidad, utilizaremos una tabla para determinar tipos de variables, aunque ya tenemos una
# nocion gracias a info(), la cantidad de nulos con su porcentaje y los valores unicos de las distintas columnas. 
# Para la Columna MRUN, deberiamos tener que todas las filas son unicos, tal como lo revisamos anteriormente, pero 
# en el caso de DEPENDENCIA, RAMA_EDUCACIONAL o CODIGO_REGION_EGRESO, deberemos ver un conjunto de valores que se repiten
# ya que estas columnas traen informacion de agrupacion. 

resumen_variables_trabajo = pd.DataFrame({
    "Tipo": inscritos[variables_trabajo].dtypes, 
    "Nulos": inscritos[variables_trabajo].isna().sum(),
    "% Nulos": inscritos[variables_trabajo].isna().mean()*100,
    "Valores Unicos": inscritos[variables_trabajo].nunique()
})

resumen_variables_trabajo # Lo muestra como tabla, mejor formato. 
# or 
#print(resumen_variables_trabajo)

,Tipo,Nulos,% Nulos,Valores Unicos
MRUN,int64,0,0.000000,320087
CLEC_REG_ACTUAL,str,0,0.000000,63
MATE1_REG_ACTUAL,str,0,0.000000,63
MATE2_REG_ACTUAL,str,0,0.000000,52
DEPENDENCIA,str,0,0.000000,7
RAMA_EDUCACIONAL,str,463,0.144648,9
CODIGO_REGION_EGRESO,str,0,0.000000,17
NOMBRE_REGION_EGRESO,str,1211,0.378335,16


La revisión anterior nos confirma la idea que antes ya habiamos respondido, `MRUN` es unica y se puede asociar a un alumno en particular. 
Aun cuando se asumia, CLEC_REG_ACTUAL, MATE1_REG_ACTUAL, no tienen valores nulos ya que es un requisito de la prueba. 
`validar con Daniela como es esta parte en la prueba, como ahora se da dos veces hay algunos modulos que se dan obligatorios y otros opcionales y se pueden utilizar puntajes de rendiciones anteriores?`
DEPENDENCIA mantiene misma condicion de no nulo, ya que hace referencia a que tipo de Establecimiento Educacional asistio el alumno. El sistema de admision requiere una institucion para poder rendir prueba. 
Para `RAMA_EDUCACIONAL` ya aparecen valores nulos, lo cual parece contraintuitivo, se supone que los establecimientos educacionales deben tener una orientacion especifica, mas adelante revisaremos el detalle y validaremos en base a documento e investigación como es que es posible tener NAs
En CODIGO_REGION_EGRESO nuevamente tenemos 0 no nulos pero extrañamente NOMBRE_REGION_EGRESO si tiene valores nulos, los cuales tendremos que analisar en contexto para revisar si es un error que trae el dataset, olvido de quien administra esto o un error al traer la informacion. 
Dado que hemos detectado situaciones, separaremos las variables en cuantitativas, puntajes particularmente y cualitativas, que tienen relacion con los atributos en terminos de categoria de las distintas variables. 

In [ ]:
# Partiremos con las numericas, obteniendo Media (promedio), Mediana y Moda para los puntajes
# evaluando ademas la existencia de valores atipicos. 

#print("Media: ", inscritos["CLEC_REG_ACTUAL"].mean())
# En esta ejecución nos encontramos con el primer problema, estamos pidiendo obtener el promedio pero la variable 
# indica ser un str (un string - cadena de caracteres) 
# Tal como se indicaba anteriormente, para revisar hacemos
inscritos["CLEC_REG_ACTUAL"].head(20)

0     597
1       0
2     467
3     455
4     841
5     576
6     607
7     467
8     455
9     525
10    782
11    860
12    860
13    650
14    555
15    397
16    690
17    500
18    607
19    756
Name: CLEC_REG_ACTUAL, dtype: str

En la ejecución anterior vemos que efectivamente existen numeros en la columna y que pareciera existir una inconsistencia entre lo que se declara en PDF de apoyo vs lo que pandas interpreta. Puede incluso deberse al tipo de "formato" en el que se guardó el dato. 
Por lo tanto, con la ayuda de un par de funciones, revisaremos que mas hay dentro de esta columna, que sabemos por los análisis anteriores, no tiene valores no nulos, es decir, todas las filas tienen algo. 

In [ ]:
CLEC_REG_ACTUAL_2_NUM = pd.to_numeric (
    inscritos["CLEC_REG_ACTUAL"],
    errors="coerce"
)
CLEC_REG_ACTUAL_2_NUM
# Revisamos si ahora podemos obtener la media, mediana, moda y definir algun criterio para valores atipicos.
print("Media: ", CLEC_REG_ACTUAL_2_NUM.mean())
print("Mediana: ", CLEC_REG_ACTUAL_2_NUM.median())
print("Moda: ", CLEC_REG_ACTUAL_2_NUM.mode())

Media:  508.96112325385656
Mediana:  576.0
Moda:  0    0.0
Name: CLEC_REG_ACTUAL, dtype: float64


Como Moda = 0, hacemos un contador de ceros sobre la columna para evaluar pertinencia

In [38]:
(CLEC_REG_ACTUAL_2_NUM == 0).sum()

np.int64(49518)

Este dato es bastante interesante ya que nos muestra que esta prueba podria no necesariamente ser un requisito, es decir, podria ser rendida o no. Lo que para el proposito de nuestra pregunta vale la pena considerar ya que quizas deberiamos mirar una variable final en vez de una situacional. 

La funcion pd.to_numeric() intenta convertir cada valor en numerico y con el argumento errors="coerce" reemplaza todo valor no numerico por un NaN, entonces si en la instruccion anterior tuvieramos otro tipo de valor que no fuera un numero o numero alike, habriamos visto incrementada las observaciones de NaN

In [19]:
inscritos["RAMA_EDUCACIONAL"].value_counts(dropna=False)

RAMA_EDUCACIONAL
H1     216909
T2      30721
T1      24819
T3      22147
H2      17760
T4       3820
H3       1993
H4        744
T5        711
NaN       463
Name: count, dtype: int64

In [20]:
inscritos["DEPENDENCIA"].value_counts(dropna=False)

DEPENDENCIA
3    168231
2     48543
4     35248
6     34372
1     19922
5     10571
       3200
Name: count, dtype: int64

In [21]:
inscritos["NOMBRE_REGION_EGRESO"].value_counts(dropna=False)

NOMBRE_REGION_EGRESO
REGION METROPOLITANA DE SANTIAGO                    123170
REGION DE VALPARAISO                                 32264
REGION DEL BIOBIO                                    29035
REGION DEL MAULE                                     20046
REGION DE LA ARAUCANIA                               18910
REGION DEL LIBERTADOR GENERAL BERNARDO O'HIGGINS     17606
REGION DE COQUIMBO                                   15053
REGION DE LOS LAGOS                                  14503
REGION DE ANTOFAGASTA                                11002
REGION DE ÑUBLE                                       9028
REGION DE LOS RIOS                                    7067
REGION DE TARAPACA                                    6658
REGION DE ATACAMA                                     5213
REGION DE ARICA Y PARINACOTA                          4629
REGION DE MAGALLANES Y DE LA ANTARTICA CHILENA        2639
REGIÓN DE AYSEN                                       2053
NaN                                